src/App.test.jsx

```diff
@@ -2,27 +2,32 @@ import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import App from './App';
 import { EntryProvider } from './context/EntryContext';
+import { ThemeProvider } from './context/ThemeContext';
 import { UserProvider } from './context/UserContext';
 
 test('renders the header', () => {
   render(
-    <EntryProvider>
-      <UserProvider>
-        <App />
-      </UserProvider>
-    </EntryProvider>
+    <ThemeProvider>
```

Comment:
One cool thing you can do is add this render block into a function, and then just call the function for each test where you need to run an identical render. 
`function renderGuestbook() {
  return render(
    <ThemeProvider>
      <EntryProvider>
          <UserProvider>
            <App />
          </UserProvider>
      </ThemeProvider>
    </EntryProvider>
  );
}`

Then inside your test... just call 
`renderGuestbook()`